srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.md

```diff
@@ -749,25 +751,24 @@ struct {
 The contents of the fields in this message are as follows:
 
 ekt\_key\_value
-: The EKT Key that the recipient should use when generating EKTCiphertext
+: The EKTKey that the recipient should use when generating EKTCiphertext
 values
 
 srtp\_master\_salt
 : The SRTP Master Salt to be used with any Master Key encrypted with this EKT
 Key
 
 ekt\_spi
-: The SPI value to be used to reference this EKT Key and SRTP Master Salt in
+: The SPI value to be used to reference this EKTKey and SRTP Master Salt in
 EKT tags (along with the EKT cipher negotiated in the handshake)
 
 ekt\_ttl
-: The maximum amount of time, in seconds, that this EKT Key can be used.  The
+: The maximum amount of time, in seconds, that this EKTKey can be used.  The
 ekt\_key\_value in this message MUST NOT be used for encrypting or decrypting
 information after the TTL expires.
 
 If the server did not provide a supported\_ekt\_ciphers extension in
-its ServerHello, then EKTKey messages MUST NOT be sent by either the
-client or the server.
+its ServerHello, then EKTKey messages MUST NOT be sent by the sever.
```

Comment:
any reason to remove the restriction on the client not sending them? that seems like it might be wrong

Reply:
@fluffy In dtls-srtp handshake, client never sends EKTKey messages, isn't it ?

Reply:
taking that back . Let me revert it to the original text.